File: fathersDay/math/Day3 (168. Excel Sheet Column Title)/Solution.py
class Solution:
    def ExcelColumnTitle(self, n: int) -> str:
        result = []
        while n > 0:
            n -= 1
            remainder = n % 26
            result.append(chr(remainder + ord('A')))
            n //= 26
        return ''.join(reversed(result))
# Example usage
solution = Solution()
print(solution.ExcelColumnTitle(1)) 

File: fathersDay/math/Day3 (168. Excel Sheet Column Title)/Solution.java
class Solution {
    public String convertToTitle(int n) {
        StringBuilder result = new StringBuilder();
        while (n > 0) {
            n--; // Adjust n to be 0-indexed (A=0, B=1, ..., Z=25)
            int remainder = n % 26;
            result.append((char)('A' + remainder));
            n /= 26;
        }
        return result.reverse().toString();
    }
}
